File: 03-0321/01-Metropolis/Metropolis.c
#include <stdio.h>

int main() {
    int isCapital, isMetropolis, citizen, bourgeois;

    printf("Is the city a capital? (capital:1 non-capital:0) ");
    scanf("%d",&isCapital);
    printf("\nPopulation? (in thousands) ");
    scanf("%d",&citizen);
    printf("\nPopulation? (in thousands) ");
    scanf("%d",&bourgeois);

    isMetropolis = ( isCapital && citizen >= 1000) || (bourgeois >= 500);

    printf("\nMetropolis : ");
    if (isMetropolis) printf("true");
    else printf("false");

    return 0;
}

File: 03-0321/01-Metropolis/Metropolis.java
import java.util.Scanner;
public class Metropolis {
    public static void main (String[] args) {
        boolean isCapital, isMetropolis;
        int citizen;
        int bourgeois;

        Scanner sc = new Scanner(System.in);
        System.out.println("Is the city a capital? (capital:1 non-capital:0) ");
        isCapital = (sc.nextInt() == 1);
        System.out.println("Population? (in thousands) ");
        citizen = sc.nextInt();
        System.out.println("bourgeois? (in thousands) ");
        bourgeois = sc.nextInt();

        isMetropolis = (isCapital && citizen >= 1000) || (bourgeois >= 500);

        System.out.println("Metropolis : "+ isMetropolis);
    }
}